Improve wording in the readme (#52)
php-parallel-lint/PHP-Parallel-Lint: README.md

[![Build Status](https://travis-ci.org/php-parallel-lint/PHP-Parallel-Lint.svg?branch=master)](https://travis-ci.org/php-parallel-lint/PHP-Parallel-Lint)
[![License](https://poser.pugx.org/php-parallel-lint/php-parallel-lint/license.svg)](https://packagist.org/packages/php-parallel-lint/php-parallel-lint)

This application checks syntax of PHP files in parallel.
It can output in plain text, colored text, json and checksyntax formats.

> Additionally `git blame` can be used to show commits that introduced the breakage.

Reviewer: ```suggestion
Additionally `blame` can be used to show commits that introduced the breakage.
```

`git blame` suggests that you'd have to run `git blame` after running this tooling, while it is actually build in via the `--blame` command-line option.

Running parallel jobs in PHP is inspired by Nette framework tests.

> The application is known to work with PHP 5.4 to 7.4

Reviewer: ```suggestion
The application is officially supported for use with PHP 5.4 to 8.0.
```

PHP 8.0 support has been around for a while now. See #5 and #41.

## Table of contents

1. [Installation](#installation)
2. [Example output](#example-output)
3. [History](#history)
4. [Command line options](#command-line-options)
5. [Recommended excludes for Symfony framework](#recommended-excludes-for-symfony-framework)
6. [Create Phar package](#create-phar-package)
7. [How to upgrade](#how-to-upgrade)

## Installation

Install with `composer` as development dependency:

    composer require --dev php-parallel-lint/php-parallel-lint

> Alternatively you can install as standalone `composer` project:

Reviewer: ```suggestion
Alternatively you can install as a standalone `composer` project:
```

![Example use of tool with error](/tests/examples/example-images/use-error.png?raw=true "Example use of tool with error")

## History

- [`php-parallel-lint/php-parallel-lint`] is a fork of [`grogy/php-parallel-lint`]
- [`grogy/php-parallel-lint`] is a fork of [`jakub-onderka/php-parallel-lint`]

[`php-parallel-lint/php-parallel-lint`]: https://github.com/php-parallel-lint/PHP-Parallel-Lint
[`grogy/php-parallel-lint`]: https://github.com/grogy/PHP-Parallel-Lint

> [`jakub-onderka/php-parallel-lint`]: https://github.com/JakubOnderka/PHP-Parallel-Lint

Reviewer: The `grogy` fork was only very briefly in play before the `php-parallel-lint` organisation was created.

If this is going to be called "History", may I suggest the following ?

> This project was originally created by @JakubOnderka and released as [`jakub-onderka/php-parallel-lint`].
> Since then, Jakub has moved on to other interests and as of January 2020, the second most active maintainer @grogy has taken over maintenance of the project and given the project - and related dependencies - a new home in the [PHP Parallel Lint](https://github.com/php-parallel-lint) organisation.
>
> It is strongly recommended for existing users of the (unmaintained) [`jakub-onderka/php-parallel-lint`] package to switch their dependency to [`php-parallel-lint/php-parallel-lint`].

Author: @jrfnl yes, I thought history is more appropriate than Fork, but I didn't spend time to find actual history, hoped the community can do that if they find it important.

I'll incorporate your suggestion now.

- `-V, --version`   		Display this application version.

## Recommended excludes for Symfony framework

> To run from command line:

Reviewer: ```suggestion
To run from the command line:
```
